README.md

```diff
@@ -1,12 +1,47 @@
 # js-releases
+
 ## Download packages from releases.hashicorp.com
 
 js-releases is a handy tool for downloading and verifying packages from releases.hashicorp.com. You can:
  - fetch metadata for a given release (or latest)
  - download the package
  - verify the SHASUM and signature
  - unpack to a specified directory
- 
+
+## Usage
+
+```
+import { Release, getRelease } from '@hashicorp/js-releases';
+
+// Setting a user agent string is optional but helpful!
+const userAgent = `Terraform-VSCode/2.1.0 VSCode/1.55.2`;
```

Comment:
```suggestion
const userAgent = `Example-Program/1.0.0 js-releases/dev`;
```
Just so that we can avoid people skewing our stats by copy pasting the code and running it 😄 

Relatedly I wonder if we should actually be appending js-releases with version in the extension?